web/src/stories/index.js

```diff
@@ -32,3 +32,30 @@ storiesOf('Buttons', module).add('various', () => (
     <Btn kind="outline" /> <Btn kind="outline" size="big" />
   </div>
 ));
+
+storiesOf('Numeric inputs', module).add('Dollars', () => {
+  class Dollar extends Component {
+    state = { value: 0 };
```

Comment:
I'm a little confused about this pattern. I'm used to seeing component state initialized in a constructor -- can you help me understand why this is different?

Reply:
Oops, I didn't even mean to include this.  😆 I just threw it in so I could modify and test them faster than reloading the whole app.  But maybe it's worth keeping, in which case I'll clean it up.

But to your question, it's just less code.  AFAIK it works the same, it's just more declarative.  Putting the state declaration in the constructor is one extra hoop you have to jump through to understand the structure of the class.  It's a [stage-3 proposal](https://github.com/tc39/proposal-private-methods) so it should make it into Javascript officially in the near future.